aries-site/src/components/icons/components/index.js

```diff
@@ -0,0 +1,4 @@
+export * from './IconControls';
+export * from './IconInput';
+export * from './IconMedia';
```

Comment:
I don't see this Icon being used anywhere, is it on the pipeline?

Reply:
It's in designs, but we don't have a page for the content yet. We can remove it for now until we have the accompanying page. Would that be best? @ShimiSun

Reply:
I think so, yes. let's keep the code lean.